src/Components/Components/src/Forms/EditContext.cs

```diff
@@ -13,6 +13,12 @@ namespace Microsoft.AspNetCore.Components.Forms
     /// </summary>
     public sealed class EditContext
     {
+        public static string ModifiedClass = "modified";
```

Comment:
Why don't you use constants?
And do they have to be `public`? `private` would be enough, as long no use outside this class is planned.

BTW: If you use `static readonly` the JIT can treat them es constants. Anyway, here `const` would be the better way.

Reply:
@gfoidl The point of @kennywangjin's PR here is to make these values mutable, hence not `const`.

I'm not sure we'll use this approach, but hopefully it explains why the PR is as it is. It's quite likely that we'd move the responsibility for this into `EditForm` and have it configurable on a per-instance basis instead.

Reply:
> hopefully it explains why the PR is as it is

Yes, thanks! 
(Unfortunately there's no reference to any issue, etc. in the description of the PR, and the boilerplate "Summary of the changes (Less than 80 chars)" distracted me.)

> have it configurable on a per-instance basis instead

Seems reasonable.